include/dlaf/solver/triangular.h

```diff
@@ -140,7 +140,7 @@ void triangular(comm::CommunicatorGrid grid, blas::Side side, blas::Uplo uplo, b
     else {
       if (op == blas::Op::NoTrans) {
         // Left Upper NoTrans
```

Comment:
IMHO, this kind of comment was useful when there wasn't the implementation, but now it is clear from the call.